crates/wasmcloud-control-interface/src/generated/ctliface.rs

```diff
@@ -188,8 +188,9 @@ pub struct HostInventory {
 pub struct ActorDescription {
     #[serde(rename = "id")]
     pub id: String,
-    #[serde(rename = "image_refs")]
-    pub image_refs: Vec<String>,
+    #[serde(rename = "image_ref")]
```

Comment:
:cry: I can't believe you're removing my plural and making it singular again. All that hard work down the drain.